apps/user/interfaces/user.interface.ts

```diff
@@ -48,7 +69,7 @@ export interface UsersProfile {
     username?: string;
   }
   
-  export interface userInfo {
+  export interface UserInfo {
```

Comment:
Let's follow naming convention for interfaces.

Reply:
Resolved